.github/workflows/CI.yml

```diff
@@ -380,6 +382,19 @@ jobs:
         with:
           python-version: '3.11'
 
+      - name: Build latest libva
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        timeout-minutes: 5
+        run: |
+          gh release download --archive=tar.gz --repo=intel/libva
+          tar xzf libva-*.tar.gz && rm libva-*.tar.gz
+          cd libva-*
+          ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu --enable-x11 --without-legacy
```

Comment:
That's probably cleaner indeed. In fact I only added x11 explicitly because the configure script failed to autodetect it due to two missing dependencies that I then had to add. For the legacy stuff I'm not 100% sure but from quickly looking into it I don't think that it's relevant. Also I'm not sure about the glx part. To be honest there's some guesswork involved...